src/renderers/WebGLRenderer.js

```diff
@@ -1164,15 +1164,19 @@ function WebGLRenderer( parameters ) {
 
 	};
 
-	function projectObject( object, camera, sortObjects ) {
+	function projectObject( object, camera, groupOrder, sortObjects ) {
 
 		if ( object.visible === false ) return;
 
 		var visible = object.layers.test( camera.layers );
 
 		if ( visible ) {
 
-			if ( object.isLight ) {
+			if ( object.isGroup ) {
+
+				groupOrder = object.renderOrder;
```

Comment:
You are passing in `groupOrder` and reassigning the value if it is a group???

Reply:
Yeah, it will apply to its children only.

Reply:
So, what is `groupOrder` if it's not a Group?